dynamic_rest/viewsets.py

```diff
@@ -336,5 +336,21 @@ def list_related(self, request, pk=None, field_name=None):
         return Response(serializer.data)
 
 
+class BulkCreationMixin(object):
+    """
+    Either create a single or many model instances in bulk by using the
+    Serializer's many=True ability from Django REST >= 2.2.5.
+    """
+    def create(self, request, *args, **kwargs):
+        if not isinstance(request.data, list):
```

Comment:
Done. However, `get_model`, `get_name` and `get_plural_name` methods have to be class methods then.